install/deployment_yamls/k8s/meshery-cpx-deployment.yaml

```diff
@@ -1,5 +1,5 @@
 
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
I am not sure if we are good to remove the beta extensions, else the PR looks great @AdheipSingh !

Reply:
@dheerajng will you chime-in on this this one?

Reply:
Helm is using apps/v1, so its good to maintain a standardised apiVersion for all the yamls.

Reply:
The version of Kubernetes v1.16 has been migrated the deployment to `app/v1` if I do not get mistakes.

Reply:
These changes are fine. We should indeed have apiVersion as apps/v1.

Reply:
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

You can check the following 

> Deployment in the extensions/v1beta1, apps/v1beta1, and apps/v1beta2 API versions is no longer served
Migrate to use the apps/v1 API version, available since v1.9. Existing persisted data can be retrieved/updated via the new version.